### PR TITLE
Added the ability to log to standard out and standard error

### PR DESCRIPTION
**Pull request by jessemapel**
Also updated the README with more up to date information and how to enable logging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,33 +52,48 @@ specification is located on
...
-`git submodule update --init --recursive` after cloning. The library can also be
-compiled against an installed versions of the CSM API by setting the BUILD_CSM
-flag to OFF during cmake configuration.
+the source code. The other three dependencies, The Abstraction Library for
+Ephemerides, the CSM API library, and googletest are included as git submodules.
+When you clone this library make sure you add the `--recursive` flag to your
+`git clone` command. Alterntively, you can run
+`git submodule update --init --recursive` after cloning.
+
+You can also install of the build requirements using Conda with the provided
```
Comment (scsides) [inline README.md]: I removed the "of"

File: README.md [hunk 2]
```diff
@@ -42,33 +52,48 @@ specification is located on
...
+## Building USGSCSM
+
+USGSCSM uses a standard cmake build system. To compile the library, and
 tests use the following commands:
 
 1. `mkdir build && cd build`
 2. `cmake .. && cmake --build .`
 
-## Testing CSM-CameraModel
+If you are using external dependencies via Conda or system level installations
```
Comment (kberryUSGS) [inline README.md:79]: Thanks for adding these! The last time I built USGSCSM I ended up opening the Makefile to remind me what the flags were named.

**Review by kberryUSGS — APPROVED** [review]
:+1: Looks great!

**Review by scsides — APPROVED** [review]
Fixed a couple typos in the github editor

**Pull request merged**